analyzer/src/funTest/kotlin/managers/PubFunTest.kt

```diff
@@ -63,8 +63,6 @@ class PubFunTest : WordSpec() {
                     val vcsPath = vcsDir.getPathToRoot(workingDir)
                     val expectedResult = patchExpectedResult(
                         expectedResultFile,
-                        custom = mapOf("pub-project" to "pub-${workingDir.name}"),
```

Comment:
Could you briefly explain in the commit message *why* they are unnecessary? Like, does e.g. the "pub-project" string simply not occur in the files?